data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1235 @@
+# Copyright (c) 2023 by Saugia
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+
+mission "Passive-Aggressive Observations"
+	invisible
+	landing
+	destination "Vigales"
+	to offer
+		has "First Contact: Gegno Scin: offered"
+		not "Gegno Genocide Defense: offered"
+	on complete
```

Comment:
No, because this spawns npcs to start with. Those npcs are the signifiers that the mission has activated, and are npcs that wouldn't otherwise be present, especially the interceptor. They are staying, so they don't leave, but depending on what the player does on the planet, when they "land", that's when the other npc fleets are taken into consideration.